gef.py

```diff
@@ -11063,7 +11071,7 @@ def selected_arena(self, value: GlibcArena) -> None:
         return
 
     @property
-    def arenas(self) -> Union[List, Iterator[GlibcArena]]:
+    def arenas(self) -> list | Iterator[GlibcArena]:
```

Comment:
If you want to use `|` I think there are many more cases where we can get rid of `Union`